scripts/l10n/json_builder.py

```
@@ -0,0 +1,115 @@
+        continue
+
+    lang = lang_folder.name
+    lang_parts = set(lang.lower().split('_'))
+
+    for po_file in lc_messages.glob("*.po"):
+        country_code = po_file.stem
+
+        try:
+            po = polib.pofile(str(po_file), encoding="utf-8")
+        except Exception:
+            continue
```

Ruff (0.15.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)